auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -160,8 +162,15 @@ public ServerResponse create() {
      */
     @Override public ServerResponse list() {
       List<Policy> policyList = null;
+      Map<String, Object> parameters = new HashMap<>();
+      if (requestBody.containsKey("OnlyAttached")) {
+        parameters.put("OnlyAttached", requestBody.get("OnlyAttached"));
+      }
+      if (requestBody.containsKey("PathPrefix")) {
```

Comment:
PathPrefix should be a constant as this is being used more than once

Reply:
fixed